extrafiles/tooling/CppClangFormat.py

```diff
@@ -47,7 +47,7 @@ def findClangFormat():
 
 			return candidate
 
-	raise FileNotFoundError("clang-format binary not found. We searched for:\n " + "\n ".join(allowed))
```

Comment:
Is this change necessary?

Reply:
probably not, but I am unsure, it was in the old commits.

Reply:
Then please drop this change, it's out of the scope of the PR.